.maestro/sync_flows/1_-_create_account.yaml

```diff
@@ -0,0 +1,13 @@
+appId: com.duckduckgo.mobile.android.debug
+---
+- launchApp:
+    clearState: true
+    stopApp: true
+- runFlow: "open_sync.yaml"
+- tapOn:
+      id: "com.duckduckgo.mobile.android.debug:id/trailingSwitch"
+- tapOn: "Turn On Sync"
+- tapOn: "Not Now"
+- tapOn: "Not Now"
```

Comment:
don't you need add some `assertVisible` in between each `tapOn: "Not Now"`? I'd guess that there's a risk of double tapping the button of one step, intead of moving two steps.